packages/react-components/react-overflow/stories/Overflow/CustomComponent.stories.tsx

```diff
@@ -0,0 +1,80 @@
+import {
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31292&commitId=717952ed69dbdcdca9d323580743ebce90ca7dda&env=prod&clientType=FLUENTUI)


  <details><summary><b>Positioning</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Positioning.auto size overflow padding - RTL.chromium.png|0| Added|
|Positioning.auto size overflow padding shorthand.chromium.png|0| Added|
|Positioning.auto size overflow padding.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>